pkgs/applications/video/vdr/xineliboutput/default.nix

```diff
@@ -1,10 +1,10 @@
 { stdenv, fetchurl, lib, vdr
 , libav, libcap, libvdpau
-, xineLib, libjpeg, libextractor, mesa, libGLU
+, xineLib, libjpeg, libextractor, libglvnd, libGLU
 , libX11, libXext, libXrender, libXrandr
 , makeWrapper
 }: let
-  name = "vdr-xineliboutput-2.1.0";
+  name = "vdr-xineliboutput-2.2.0";
```

Comment:
Let's break this up into `pname` and `version` in the derivation. This is the new standard.